docker/connect/connect_start.sh

```diff
@@ -0,0 +1,131 @@
+# bin/bash
+DOCKER_HOST_IP = $1
+
+echo "------------------------------------------"
+echo "Kafka connector creation started."
+echo "Provided Docker Host IP: $DOCKER_HOST_IP"
+echo "------------------------------------------"
+
+
+declare -A OdeRawEncodedBSMJson=([name]="topic.OdeRawEncodedBSMJson" [collection]="OdeRawEncodedBSMJson"
+    [convert_timestamp]=false [timefield]="")
+declare -A OdeBsmJson=([name]="topic.OdeBsmJson" [collection]="OdeBsmJson"
```

Comment:
I don't think topics that are created by the ODE or the Geojson converter (ones prefixed with 'Ode') need to be archived by this app.

Reply:
I believe that we are planning on archiving these, as they are just a JSON object wrapper to the ASN.1 input messages that we are wanting to store. We are also wanting to store the processed messages from the GeoJson converter for querying.